lib/kitchen/verifier/shell.rb

```diff
@@ -0,0 +1,102 @@
+# -*- encoding: utf-8 -*-
+#
+# Author:: SAWANOBORI Yukihiko (<sawanoboriyu@higanworks.com>)
+#
+# Copyright (C) 2015, HiganWorks LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "kitchen/verifier/base"
+
+module Kitchen
+
+  module Verifier
+
+    # Shell verifier for Kitchen. This verifier just execute shell command from local.
+    #
+    # @author SAWANOBORI Yukihiko (<sawanoboriyu@higanworks.com>)
+    class Shell < Kitchen::Verifier::Base
+      require "mixlib/shellout"
+
+      kitchen_verifier_api_version 1
+
+      plugin_version Kitchen::VERSION
+
+      default_config :sleep, 0
+      default_config :command, "true"
+      default_config :shellout_opts, {}
+      default_config :live_stream, $stdout
+      default_config :remote_exec, false
+
+      # (see Base#call)
+      def call(state)
+        info("[#{name}] Verify on instance=#{instance} with state=#{state}")
+        sleep_if_set
```

Comment:
Some services need initializing itself when first started. Such as jenkins, pentaho-ce and etc. I want to wait few seconds before start test.